src/service/cli/generate.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const fs = require(`fs`);
+const {
+  getRandomInt,
+  shuffle,
+} = require(`../../utils`);
+
+const SENTENCE_MAX_LENGTH = 5;
+const FILE_NAME = `mocks.json`;
+const Amount = {
+  DEFAULT: 1,
+  MAX: 1000
+};
+
+const TITLES = [
+  `Продам книги Стивена Кинга.`,
+  `Продам новую приставку Sony Playstation 5.`,
+  `Продам отличную подборку фильмов на VHS.`,
+  `Куплю антиквариат.`,
+  `Куплю породистого кота.`,
+  `Продам коллекцию журналов «Огонёк».`,
+  `Отдам в хорошие руки подшивку «Мурзилка».`,
+  `Продам советскую посуду. Почти не разбита.`,
+  `Куплю детские санки.`
+];
+
+const SENTENCES = [
+  `Товар в отличном состоянии.`,
+  `Пользовались бережно и только по большим праздникам.`,
+  `Продаю с болью в сердце...`,
+  `Бонусом отдам все аксессуары.`,
+  `Даю недельную гарантию.`,
+  `Если товар не понравится — верну всё до последней копейки.`,
+  `Это настоящая находка для коллекционера!`,
+  `Если найдёте дешевле — сброшу цену.`,
+  `Таких предложений больше нет!`,
+  `Две страницы заляпаны свежим кофе.`,
+  `При покупке с меня бесплатная доставка в черте города.`,
+  `Кажется, что это хрупкая вещь.`,
+  `Мой дед не мог её сломать.`,
+  `Кому нужен этот новый телефон, если тут такое...`,
+  `Не пытайтесь торговаться. Цену вещам я знаю.`
+];
+
+const CATEGORIES = [
+  `Книги`,
+  `Разное`,
+  `Посуда`,
+  `Игры`,
+  `Животные`,
+  `Журналы`,
+];
+
+const OfferType = {
+  OFFER: `offer`,
+  SALE: `sale`,
+};
+
+const SumRestrict = {
+  MIN: 1000,
+  MAX: 100000,
+};
+
+const PictureRestrict = {
+  MIN: 1,
+  MAX: 16
+};
+
+const getPictureFileName = (int = 0) => {
+  if (int < 10) {
+    int = `0${int}`;
+  }
+  return `item${int}.jpg`;
+};
+
+const generateOffers = (count) => (
+  Array(count).fill({}).map(() => ({
+    title: TITLES[getRandomInt(0, TITLES.length - 1)],
+    picture: getPictureFileName(getRandomInt(PictureRestrict.MIN, PictureRestrict.MAX)),
+    description: shuffle(SENTENCES).slice(0, SENTENCE_MAX_LENGTH).join(` `),
+    type: OfferType[Object.keys(OfferType)[Math.floor(Math.random() * Object.keys(OfferType).length)]],
+    sum: getRandomInt(SumRestrict.MIN, SumRestrict.MAX),
+    category: shuffle(CATEGORIES).slice(0, getRandomInt(1, CATEGORIES.length - 1))
+  }))
+);
+
+
+module.exports = {
+  name: `--generate`,
+  run(args) {
+    const [count] = args;
+    const offersCount = Number.parseInt(count, 10) || Amount.DEFAULT;
+
+    if (offersCount > Amount.MAX) {
+      console.log(`Можно сгенерировать не больше ${Amount.MAX} объявлений.`);
+      process.exit();
+    }
+
+    const content = JSON.stringify(generateOffers(offersCount));
+
+    fs.writeFile(FILE_NAME, content, (err) => {
+      if (err) {
+        return console.error(err);
+      }
+      return console.info(`Файл с объявлениями ${FILE_NAME} создан. Количество объявлений: ${offersCount}.`);
```

Comment:
```suggestion
      console.info(err ? err : `Файл с объявлениями ${FILE_NAME} создан. Количество объявлений: ${offersCount}.`);
```
Есть ли в **node.js** разница между `console.log/error/warn/info`?

Reply:
да, log, info выводят сообщение в stdout
а error и warn в stderr